### DISCUSSION TITLE
Make class names selectable in your website

### DISCUSSION THREAD

**Discussion by kodmanyagha** [FortAwesome/Font-Awesome#20955] [Feature Requests]
Hello. I love your project. I can't able to think a project without fontawesome. This is really awesome project. I want a small thing from you. Please look that picture:

![fontawesome](https://user-images.githubusercontent.com/3948692/74931500-c45e6b00-53f0-11ea-9977-0538f1296964.png)

In here I can copy to clipboard all &lt;i&gt; tag when I click to it. But this is classic html style. I need only class names not all html &lt;i&gt; tag. Becouse I'm using reactjs and it doesn't using "class" attribute. And I'm adding different class names to &lt;i&gt; tag for example "nav-icons" (&lt;i className="nav-icon fas fa-landmark" /&gt;)

When I want to use different icon then I'm coping existing icon (&lt;i className="nav-icon fas fa-landmark" /&gt;), paste to needed place in my project and I want to change only "fas fa-landmark" section to "fas fa-money-bill". For this I need only class names, not all &lt;i&gt; tag. Please make selectable class names section on your website.

I know you're busy, you have lots of things to do but I think this is important lots of people and me :)

Thanks.

**Comment by ackava** [comment]
This is by far the most important feature when we are using some framework, where we want to use class name in an expression such as ..

```typescript
    <RedButton class={ someCondition ? "fas fa-landmark" : "fas fa-money-bill"  } />
```

We can't copy and paste entire tag and remove all the surrounding. It doesn't seem too big of a feature.

**Comment by ackava** [comment]
@tagliala Is there any reason your team does not want to implement this simple feature? I am pro member since two years and I have sent this request on support as well. The whole new layout was recently changed, is there any specific reason to not allow us to copy only class names?

**Comment by faileon** [comment]
Please. This. I cannot do this anymore. You know the workaround I figured? Because of the autocopy feature (UX nightmare, just give us dedicated button for yanking, do not hook on the click event...) I have to drag over the text inside the icon, KEEP HOLDING left mouse button, hit ctrl+c and then `<esc>`. I can't do this anymore...
/rant

**Comment by redeye86** [comment]
+1
